x-pack/legacy/plugins/ml/public/data_frame_analytics/pages/analytics_exploration/components/regression_exploration/evaluate_panel.tsx

```diff
@@ -30,23 +39,28 @@ const rSquaredText = i18n.translate(
 );
 const defaultEval: Eval = { meanSquaredError: '', rSquared: '', error: null };
 
-export const EvaluatePanel: FC<Props> = ({ jobId, index, dependentVariable }) => {
+export const EvaluatePanel: FC<Props> = ({ jobConfig, jobStatus }) => {
   const [trainingEval, setTrainingEval] = useState<Eval>(defaultEval);
   const [generalizationEval, setGeneralizationEval] = useState<Eval>(defaultEval);
   const [isLoadingTraining, setIsLoadingTraining] = useState<boolean>(false);
   const [isLoadingGeneralization, setIsLoadingGeneralization] = useState<boolean>(false);
 
+  const index = idx(jobConfig, _ => _.dest.index) as string;
```

Comment:
Wondering why `idx` is necessary here? The type `DataFrameAnalyticsConfig` specifies `.dest.index` not as optional.